examples/htlc-pyteal-ts/README.md

```diff
@@ -22,6 +22,8 @@ This example is using PyTEAL, so make sure to follow the Python3 setup described
 
 ## Run
 
+**NOTE**: Make sure to run `yarn build` before the following commands.
```

Comment:
do we always need to run this ? if yes then, we have to update other examples as well

Reply:
I think we need it to compile typescript to javascript.
Other solution would be to use `tsnode`

Reply:
Maybe we should add more details here.

Reply:
> I think we need it to compile typescript to javascript.

Exactly. @amityadav0 we need to build because we need to compile typescript to javascript (`yarn build: tsc --build .` is using the typescript complier (tsc) ). And in deployer, if we encounter a `.ts` file, we are loading the 'compiled' file from `/build`. We don't need to update the other examples as they're already in js. 

Will add more details here, as why we're building.